graylog2-web-interface/src/logic/reactQueryKeys.ts

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright (C) 2020 Graylog, Inc.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the Server Side Public License, version 1,
+ * as published by MongoDB, Inc.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * Server Side Public License for more details.
+ *
+ * You should have received a copy of the Server Side Public License
+ * along with this program. If not, see
+ * <http://www.mongodb.com/licensing/server-side-public-license>.
+ */
+
+export const GET_INDEXER_CLUSTER_HEALTH = 'GET_INDEXER_CLUSTER_HEALTH';
+export const GET_INDEXER_CLUSTER_NAME = 'GET_INDEXER_CLUSTER_NAME';
```

Comment:
I like the idea to separate the API calls and the query keys in general, but I would suggest to move both into the component where we use `react-query`, to keep it consistent with the other cases where we use `react-query`.